### ISSUE TITLE
4 saplings tree don't grow

### ISSUE THREAD

**Issue by SirvydasS** [BentoBoxWorld/TwerkingForTrees#2]
#### Describe the bug

When I put 4 sapling to grow up a big tree and  star twerking then grow ups only one sapling tree and other 3 saplings disappear.

#### Steps to reproduce the behavior

Put 4 saplings near to each other (2x2), start twerk and wait.

#### Expected behavior

It should be a big tree from 4 saplings or at least not to lose others saplings.

#### Screenshots and videos (Optional)

https://youtu.be/KBcSbqoI3CE
##### BentoBox Version (Mandatory)

```
Running PAPER 1.14.4.
BentoBox version: 1.9.1-SNAPSHOT-b1505
Database: JSON
Loaded Game Worlds:
bskyblock_world (BSkyBlock): Overworld, Nether, End
Loaded Addons:
Biomes 1.6.0.1-SNAPSHOT-b149 (ENABLED)
BSkyBlock 1.9.0 (ENABLED)
Challenges 0.9.0-SNAPSHOT-#394 (ENABLED)
DimensionalTrees 1.6.0-SNAPSHOT-b78 (ENABLED)
Greenhouses 0.4.5-SNAPSHOT-b113 (ENABLED)
Level 1.9.1-SNAPSHOT-b328 (ENABLED)
Likes 1.7.0-SNAPSHOT-#14 (ENABLED)
Limits 1.9.0-SNAPSHOT-b186 (ENABLED)
MagicCobblestoneGenerator 1.7.1-SNAPSHOT-#86 (ENABLED)
TwerkingForTrees 1.0.1 (ENABLED)
Warps 1.9.0 (ENABLED)

```

#### Plugins (Optional)

```
_______
```

**Comment by tastybento** [comment]
I think it’s the quality of your twerking - you need more “umph” and a bit more “yaaah”. 😉

Okay, I’ll see what can be done here.

**Comment by SirvydasS** [comment]
i tested latest version. When i put one jungle sapling on 1 dirt and start twerking you get 2x2 big tree ant 3 more dirts teleports above wood.

**Comment by tastybento** [comment]
Yep, that's a bug. Please try latest build. https://ci.codemc.io/job/BentoBoxWorld/job/TwerkingForTrees/6/
ad4eb34f29b1bac80314132476a723f80579010d

**Comment by SirvydasS** [comment]
still same bug with 1.0.3 version. jungle from 1 sapling grow as 4 sapling was put

**Comment by tastybento** [comment]
Yep, fixed in 1.0.4. Bukkit API JavaDoc is wrong about the tree size. Thanks!
